src/main/java/com/minecolonies/rankup/util/PermissionsUtils.java

```diff
@@ -126,23 +119,25 @@ public Integer balanceToNextGroup(final User user)
             userMoney = 0;
         }
 
-        if (getNextGroup(getPlayerHighestRankingGroup(user)).equals(""))
+        if (getNextGroup(user).equals("") || groupsConfig == null)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1132)